pilot/cmd/pilot-agent/options/security.go

```diff
@@ -140,6 +140,11 @@ func SetupSecurityOptions(proxyConfig *meshconfig.ProxyConfig, secOpt *security.
 	if o.ProvCert != "" && o.FileMountedCerts {
 		return nil, fmt.Errorf("invalid options: PROV_CERT and FILE_MOUNTED_CERTS are mutually exclusive")
 	}
+
+	if o.FileMountedCerts && o.OutputKeyCertToDir != "" {
+		return nil, fmt.Errorf("invalid options: OUTPUT_CERTS and FILE_MOUNTED_CERTS are mutually exclusive")
```

Comment:
I think istio VM is setting OUTPUT_CERTS by default which is probably why

Reply:
Rather than the UX of:

1. I install on VMs with some special CA or file mounted certs setup
2. I get this error, I go manually remove `OUTPUT_CERTS` from the VMs config. 
3. Now it works. 

Instead maybe:

1. I install on VMs with something that triggers `o.FileMountedCerts`
2. Logs warn that outputcerts is ineffective due to this conflict (possibly the above conflict as well? i'd rather keep the change small)
3. We unset OUTPUT_CERTS here

Reply:
wdyt?

Reply:
SGTM